web/src/components/Failure.tsx

```diff
@@ -32,7 +35,7 @@ const Failure = ({ errorMessage, goToPath, goToMessage = 'goTo.start' }: Failure
       <div>
         <StyledIcon src={SadSmileyIcon} />
       </div>
-      <div role='alert'>{t(errorMessage)}</div>
+      <ErrorMsg role='alert'>{t(errorMessage)} </ErrorMsg>
```

Comment:
Try to use readable names like `ErrorMessage` instead.

Reply:
I removed the overflowing "margin" and "padding", adjusted the "gap" and finally added a margin-top on the "labels" to separate the sections a little.